server/routes/save-record.route.js

```diff
@@ -194,6 +209,17 @@ const _createSection10Body = async (request, itemType, itemDescription) => {
   }
 }
 
+const _getPhotoUrls = async (request) => {
+  const photos = await RedisService.get(request, RedisKeys.UPLOAD_PHOTO)
+  const photoUrls = {}
+  if (photos && photos.files && photos.files.length > 1) {
```

Comment:
This should be testing for greater than zero photos, not greater than one - it might be a good idea to have a test for this?